CONTRIBUTING.md

```diff
@@ -1 +1,54 @@
-Esri welcomes contributions from anyone and everyone. Please see our [guidelines for contributing](https://github.com/esri/contributing).
\ No newline at end of file
+Esri welcomes contributions from anyone and everyone. Please see our [guidelines for contributing](https://github.com/esri/contributing).
+
+## How you can contribute code to this repository
+
+### Make a new branch in the repo
+Start contributing to the *solutions-geoprocessing-toolbox* repo by making a branch. Make a new branch from **dev** on github.com: 
+
+Select the **dev** drop down
+Type a new branch name. The convention uses your initials and a title separated by dashes "-". For example: *xy-new-suitabilty-tool*
+
+![Create new branch](CreateNewBranchDialogScreenshot.png)
+
+*Fetch* and *merge* the new branch from the **Esri** remote: [Getting Changes from the repo](#getting-changes-from-the-repo).
+
+Then checkout your new branch:
+
+`git checkout <new_branch>`
+
+
+### Getting Changes from the repo
+The solutions-geoprocessing-toolbox repo changes without notice, so make sure you are getting the latest updates often.
+
+Using git command line check your *remote*:
+
+`> git remote -v`
+
+This command should return a list of remotes including:
+
+`Esri	https://github.com/Esri/solutions-geoprocessing-toolbox.git (fetch)`
```

Comment:
That formatting doesn't seem to like carriage returns for multiple lines. It took booth lines and made them one. An extra CR made it worse. I'll leave it like that for now. :(